pkg/middleware/batch.go

```diff
@@ -132,40 +175,20 @@ func BatchRouter(next http.Handler) http.Handler {
 			}
 
 			batchRes := NewBatchResponse()
-			for _, op := range req.Operations {
+			var eg errgroup.Group
```

Comment:
Unfortunately, it won't work for our case. We cannot have request's context in async calls. As soon as our controller is done with the request, it will be canceled. Technically, as soon as our client got response , then it can close the connection. It won't work for our parallel processing, especially when you have less workers than the number of operations

Reply:
we have resolved it and agreed what is a hard error vs soft error